web/app/configs/capacities.json

```diff
@@ -239,6 +245,13 @@
             "wind": 2938
         }
     },
+    "RS": {
+        "capacity": {
+            "coal": 5644,
+            "hydro": 3006,
+            "unknown": 428
```

Comment:
Perfect, we use IPCC median values, so 490 will be indeed the emission intensity used for gas. Thanks!